config.py

```diff
@@ -31,23 +36,29 @@
 update_feed = 'https://raw.githubusercontent.com/EDCD/EDMarketConnector/releases/edmarketconnector.xml'
 update_interval = 8*60*60
 
-if getenv("EDMC_NO_UI"):
+if os.getenv("EDMC_NO_UI"):
```

Comment:
This should just be able to use EDMCLogging.get_main_logger(), rather than repeating the if/else code.

Reply:
Do you want me to change that in this PR?

Reply:
Yes please, it should be a minimal change.

Reply:
Creates a circular import (which is probably why config doesn't import anything else 

```
Traceback (most recent call last):
  File "./EDMarketConnector.py", line 19, in <module>
    from config import applongname, appname, appversion, appversion_nobuild, config, copyright
  File "/home/ad/development/python/EDMarketConnector/config.py", line 24, in <module>
    import EDMCLogging
  File "/home/ad/development/python/EDMarketConnector/EDMCLogging.py", line 49, in <module>
    from config import appcmdname, appname, config
ImportError: cannot import name 'appcmdname' from partially initialized module 'config' (most likely due to a circular import) (/home/ad/development/python/EDMarketConnector/config.py)
```

Reply:
I had an after-comment suspicion it might have been such a special case.  We should add a comment to that effect.